packages/algo-ts/src/arc4/index.ts

```diff
@@ -91,5 +91,5 @@ export function baremethod<TContract extends Contract>(config?: BareMethodConfig
  * @returns The ARC4 method selector. Eg. `02BECE11`
  */
 export function methodSelector(methodSignature: string): bytes {
-  throw new NoImplementation()
+  return ctxMgr.instance.op.sha512_256!(Bytes(encodingUtil.utf8ToUint8Array(methodSignature))).slice(0, 4)
```

Comment:
Can this import and call the function from the op namespace rather than using the context manager? ie. `import { sha512_256 } from './op.ts'` then `return sha512_256(Bytes(methodSignature)).slice(0, 4)

Reply:
I did that initially and it introduced a circular dependency.

Reply:
did you import from `../op.ts` or `../index.ts` 🤔